test/e2e/preset_test.go

```diff
@@ -33,6 +33,9 @@ const (
 	PresetMistral7BInstructModel = "mistral-7b-instruct"
 	PresetPhi2Model              = "phi-2"
 	PresetPhi3Mini128kModel      = "phi-3-mini-128k-instruct"
+	WorkspaceHashAnnotation      = "workspace.kaito.io/hash"
+	// WorkspaceRevisionAnnotation is the Annotations for revision number
```

Comment:
This is a hash right? Or is it a revision number (e.g. "1")? 

I think its a hash be
```suggestion
	// WorkspaceRevisionAnnotation represents a unique hash value of the workspace configuration to track changes
```